app/models/user.rb

```diff
@@ -256,4 +377,16 @@ def send_confirmation_instructions
   end
 
   add_method_tracer :send_devise_notification, "Custom/#{name}/send_devise_notification"
+
+  private
+
+  # Private from UserDecorator
+  def lockout_period
+    IdentityConfig.store.lockout_period_in_minutes.minutes
+  end
+
+  # Private from UserDecorator
+  def lockout_period_expired?
+    lockout_time_expiration < Time.zone.now
+  end
```

Comment:
I don't think the comments give us much, I'd remove em

```suggestion
  private

  def lockout_period
    IdentityConfig.store.lockout_period_in_minutes.minutes
  end

  def lockout_period_expired?
    lockout_time_expiration < Time.zone.now
  end
```

Reply:
Whoops, yeah, forgot to take these out...thanks!